.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
   - "node"
-  - "0.12"
-  - "0.10"
-  - "iojs"
-  - "iojs-v2.0.0"
\ No newline at end of file
+  - "4.0"
```

Comment:
Should be "4" and "6" to pick the latest 4.x and 6.x versions, rather than 4.0 and 6.0 which nobody is using anymore.